### PR TITLE
Add exclusion filters and uninstall to App Sec docs

### PR DESCRIPTION
**Pull request by sarina-dd**
### What does this PR do?
Adds a section for exclusion filters and uninstallation to the App Sec setup + config section

### Motivation
Q1 OKR

### Preview
https://docs-staging.datadoghq.com/sarina/appsec-setup-edits/security_platform/application_security/setup_and_configure/#exclusion-filters

### Additional Notes
It looks like the formatter formatted some white space for us, magic ✨ 

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/security_platform/application_security/setup_and_configure.md [hunk 1]
```diff
@@ -190,9 +190,25 @@ The data that you’re collecting with Datadog can contain sensitive information
 
 The Datadog Agent and some tracing libraries have options available to address these situations and modify or discard spans. See [APM Data Security][1] for details that also apply to Application Security.
 
+## Exclusion filters
```
Comment (sarina-dd) [inline content/en/security_platform/application_security/setup_and_configure.md:193]: @arnaudbreton Bom dia! 👋  Opened this PR and added exclusion filters / uninstall content. Exclusion filters are not billed, correct? And is there a limit on the number of filters you can set? I'll add those notes in the PR once confirmed. Feel free to add any suggestions to this area, and uninstallation as well.

File: content/en/security_platform/application_security/setup_and_configure.md [hunk 2]
```diff
@@ -190,9 +190,25 @@ The data that you’re collecting with Datadog can contain sensitive information
...
+You can set an exclusion filter, which ignore events from a rule, to eliminate these noisy signal patterns and focus on legitimate suspicious requests.
+
+To create an exclusion filter, do one of the following:
+
+- Click on a signal in [Application Security Signals][2] and click the **Create Exclusion Filter** button in the top left corner. This method automatically generates a filter query for the targeted service.
+- Navigate to [Exclusion Filters Configuration][3] and manually configure a new exclusion filter based on your own filter query.
+
+## Uninstall
+
+To uninstall Application Security, remove the `DD_APPSEC_ENABLED=true` environment variable from your application configuration. If you need additional help, contact [Datadog support].
```
Comment (sarina-dd) [inline content/en/security_platform/application_security/setup_and_configure.md:206]:
```suggestion
To uninstall Application Security, remove the `DD_APPSEC_ENABLED=true` environment variable from your application configuration. If you need additional help, contact [Datadog support][4].
```

File: content/en/security_platform/application_security/setup_and_configure.md [hunk 3]
```diff
@@ -190,9 +190,25 @@ The data that you’re collecting with Datadog can contain sensitive information
...
 
 ## Further Reading
 
 {{< partial name="whats-next/whats-next.html" >}}
 
 [1]: /tracing/setup_overview/configure_data_security/
+[2]: https://app.datadoghq.com/security/appsec/signals
+[3]: https://app.datadoghq.com/security/appsec/exclusions
```
Comment (sarina-dd) [inline content/en/security_platform/application_security/setup_and_configure.md:214]:
```suggestion
[3]: https://app.datadoghq.com/security/appsec/exclusions
[4]: /help/
```

File: content/en/security_platform/application_security/setup_and_configure.md [hunk 4]
```diff
@@ -190,9 +190,26 @@ The data that you’re collecting with Datadog can contain sensitive information
 
 The Datadog Agent and some tracing libraries have options available to address these situations and modify or discard spans. See [APM Data Security][1] for details that also apply to Application Security.
 
+## Exclusion filters
+
+There may be a time when an Application Security signal is a false positive. For example, a targeted service is repeatedly generating the same signal, but the signal has been reviewed and is not a threat.
+
+You can set an exclusion filter, which ignore events from a rule, to eliminate these noisy signal patterns and focus on legitimate suspicious requests.
```
Comment (arnaudbreton) [inline content/en/security_platform/application_security/setup_and_configure.md:197]: > There may be a time when an Application Security signal is a false positive. For example, a targeted service is repeatedly generating the same signal, but the signal has been reviewed and is not a threat.

A few language tweaks to consider:
- A service doesn't generate signals. AppSec detects signals based on traffic it receives

> You can set an exclusion filter, which ignore events from a rule, to eliminate these noisy signal patterns and focus on legitimate suspicious requests.
Ignore security events from the event rules

File: content/en/security_platform/application_security/setup_and_configure.md [hunk 5]
```diff
@@ -190,9 +190,26 @@ The data that you’re collecting with Datadog can contain sensitive information
...
+
+To create an exclusion filter, do one of the following:
+
+- Click on a signal in [Application Security Signals][2] and click the **Create Exclusion Filter** button in the top left corner. This method automatically generates a filter query for the targeted service.
+- Navigate to [Exclusion Filters Configuration][3] and manually configure a new exclusion filter based on your own filter query.
+
+## Uninstall
```
Comment (arnaudbreton) [inline content/en/security_platform/application_security/setup_and_configure.md:204]: Disable?

File: content/en/security_platform/application_security/setup_and_configure.md [hunk 6]
```diff
@@ -190,9 +190,26 @@ The data that you’re collecting with Datadog can contain sensitive information
...
+
+To uninstall Application Security, remove the `DD_APPSEC_ENABLED=true` environment variable from your application configuration. If you need additional help, contact [Datadog support][4].
```
Comment (arnaudbreton) [inline content/en/security_platform/application_security/setup_and_configure.md:206]: The service must be restarted for the change to be taken in account

File: content/en/security_platform/application_security/setup_and_configure.md (hunk 1 above)
Comment (arnaudbreton) [reply content/en/security_platform/application_security/setup_and_configure.md:193]: Indeed, requests (traces) matching an exclusion filter won't be charged. 

About the limit of filters, I don't think so but I'm asking to the team. [Slack thread](https://dd.slack.com/archives/C01UQV7HA3A/p1645012496120029)

**Review by arnaudbreton — CHANGES_REQUESTED** [review]

File: content/en/security_platform/application_security/setup_and_configure.md [hunk 7]
```diff
@@ -190,9 +190,26 @@ The data that you’re collecting with Datadog can contain sensitive information
 
 The Datadog Agent and some tracing libraries have options available to address these situations and modify or discard spans. See [APM Data Security][1] for details that also apply to Application Security.
 
+## Exclusion filters
+
+There may be a time when an Application Security signal is a false positive. For example, a targeted service is repeatedly generating the same signal, but the signal has been reviewed and is not a threat.
```
Comment (sarina-dd) [inline content/en/security_platform/application_security/setup_and_configure.md:195]:
```suggestion
There may be a time when an Application Security signal is a false positive. For example, Application Security repeatedly detects
the same suspicious request and a signal is generated, but the signal has been reviewed and is not a threat.
```

File: content/en/security_platform/application_security/setup_and_configure.md (hunk 5 above)
Comment (sarina-dd) [inline content/en/security_platform/application_security/setup_and_configure.md:204]:
```suggestion
## Disable
```

File: content/en/security_platform/application_security/setup_and_configure.md (hunk 6 above)
Comment (sarina-dd) [inline content/en/security_platform/application_security/setup_and_configure.md:206]:
```suggestion
To disable Application Security, remove the `DD_APPSEC_ENABLED=true` environment variable from your application configuration. Once removed, restart your service.

If you need additional help, contact [Datadog support][4].
```

**Comment by sarina-dd** [comment]
@arnaudbreton Resolved outstanding comments, ready for final review 👍

**Review by arnaudbreton — APPROVED** [review]

**Comment by arnaudbreton** [comment]
One minor comment and LGTM overall. Thanks!

File: content/en/security_platform/application_security/setup_and_configure.md [hunk 8]
```diff
@@ -190,9 +190,31 @@ The data that you’re collecting with Datadog can contain sensitive information
...
+You can set an exclusion filter, which ignore events from a rule, to eliminate these noisy signal patterns and focus on legitimate suspicious requests.
+
+To create an exclusion filter, do one of the following:
+
+- Click on a signal in [Application Security Signals][2] and click the **Create Exclusion Filter** button in the top left corner. This method automatically generates a filter query for the targeted service.
+- Navigate to [Exclusion Filters Configuration][3] and manually configure a new exclusion filter based on your own filter query.
+
+**Note**: Requests (traces) matching an exclusion filter are not billed.
+
+## Disable
```
Comment (sarina-dd) [inline content/en/security_platform/application_security/setup_and_configure.md:207]:
```suggestion
## Disable Application Security
```

**Review by urseberry — APPROVED** [review]

**Pull request merged**